README.md

```diff
@@ -0,0 +1,7 @@
+= UM Center for Statistical Genetics Coding Style Guideliness =
```

Comment:
I can tell you're most used to Wiki markup! README files on GitHub won't format Wiki markup properly but will cleanly format [Markdown](http://daringfireball.net/projects/markdown/syntax) automatically. Headings in Markdown can be done by starting a line with a hash:

```
# Heading
```
# Heading
```
## Subheading
```
## Subheading
Or by "underlining" a line with hyphens or equals signs:

```
Heading
=======
```
Heading
=======

```
Subheading
----------
```
Subheading
----------

I would encourage using hash characters for headings as its less verbose and generally cleaner.